ts/src/okx.ts

```diff
@@ -5886,15 +5890,22 @@ export default class okx extends Exchange {
             'ccy': currency['id'],
             'period': timeframe,
         };
-        if (since !== undefined) {
-            request['begin'] = since;
-        }
-        const until = this.safeInteger2 (params, 'till', 'until');
-        if (until !== undefined) {
-            request['end'] = until;
-            params = this.omit (params, [ 'until', 'till' ]);
+        let type = undefined;
+        let response = undefined;
+        [ type, params ] = this.handleMarketTypeAndParams ('fetchOpenInterestHistory', undefined, params);
```

Comment:
@Dan-krm we should use market whenever possible to infer the `type`

Reply:
my bad didn't notice we were using the symbol as a currency code, which is by itself a bit weird and misleading